docs/content/sre/tenant-operator/hibernation.md

```diff
@@ -45,7 +45,12 @@ spec:
 ```
 > Currently, Hibernation is available only for StatefulSets and Deployments.
 
-It is also possible to hibernate non-tenant namespaces by creating a ResourceSupervisor CR manually. This ResourceSupervisor definition will contain the hibernation cron schedule, the names of the namespaces to be hibernated, and the names of the ArgoCD AppProjects whose ArgoCD Applications have to be hibernated (as per the given schedule).
+### Manual creation of ResourceSupervisor
+
+Hibernation can also be applied by creating a ResourceSupervisor resource manually.
+The ResourceSupervisor definition will contain the hibernation cron schedule, the names of the namespaces to be hibernated, and the names of the ArgoCD AppProjects whose ArgoCD Applications have to be hibernated (as per the given schedule).
+
+This method can be used to hibernate non-tenant namespaces and ArgoCD AppProjects, or to hibernate specific namespaces and AppProjects belonging to a tenant. This method also enables cross-tenant hibernation, i.e. hibernating namespaces that belong to a tenant that the user is not a member of.
```

Comment:
user does not have hibernation permission, only cluster admin does, so this line should be removed or modified